charts/tidb-backup/templates/backup-job.yaml

```diff
@@ -45,35 +45,50 @@ spec:
           mountPath: "/savepoint-dir"
         - name: data
           mountPath: "/data"
-      {{- if .Values.backup.gcp }}
+      {{- if .Values.gcp }}
         - name: gcp-credentials
           mountPath: "/gcp"
           readOnly: true
       {{- end }}
         env:
-        - name: MY_POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-      {{- if .Values.backup.gcp }}
+        - name: BACKUP_NAME
+          value: {{ .Values.name | quote }}
+      {{- if .Values.gcp }}
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /gcp/credentials.json
       {{- end }}
+      {{- if .Values.ceph }}
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.ceph.secretName }}
+              key: access_key
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.ceph.secretName }}
+              key: secret_key
+      {{- end }}
+        - name: TIDB_USER
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.secretName }}
+              key: user
         - name: TIDB_PASSWORD
           valueFrom:
             secretKeyRef:
-              name: {{ .Values.clusterName }}-backup
-              key: user-password
+              name: {{ .Values.secretName }}
+              key: password
       restartPolicy: OnFailure
       volumes:
       - name: savepoint-dir
         emptyDir: {}
       - name: data
         persistentVolumeClaim:
-          claimName: {{ .Values.clusterName }}-backup
-    {{- if .Values.backup.gcp }}
+          claimName: {{ .Values.clusterName }}-{{ .Values.name }}
+    {{- if .Values.gcp }}
       - name: gcp-credentials
         secret:
-          secretName: {{ .Values.clusterName }}-backup
```

Comment:
in the test cases we should initialize two cluster backup and restore to use the same pvc. but the claimName has a prefix clustername, this will not suit our first need.